packages/dcos-diagnostics/extra/dcos-diagnostics-master.service

```diff
@@ -13,5 +13,6 @@ User=dcos_diagnostics
 # Allow r/w access to the socket file
 SupplementaryGroups=dcos_adminrouter
 ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-diagnostics-master
+ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-checks-master
```

Comment:
The plan going forward is to separate out the checks as a single unit, depending on the failure we can decide how to handle it.